firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -91,8 +92,8 @@ int occupant_detection_timer_set(String input) {
 
     // if e, echo the current threshold
     if (*holder == 'e') {
-        EEPROM.get(ADDR_STATE0_OCCUPANT_DETECTION_TIMER, state0_occupant_detection_timer);
-        returnFlag = state0_occupant_detection_timer / 1000;
+        EEPROM.get(ADDR_STATE0_OCCUPANT_DETECTION_TIMER, state0_occupant_detection_max_time);
```

Comment:
Reason for max_time instead of timer?

Reply:
If it's significant, could you rename the ADDR\_...\_TIMER to ADDR\_...\_MAX_TIME?